po/fr.po

```diff
@@ -0,0 +1,722 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE paru'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the paru package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: paru VERSION\n"
+"Report-Msgid-Bugs-To: https://github.com/Morganamilo/paru\n"
+"POT-Creation-Date: 2021-07-02 13:11+0000\n"
+"PO-Revision-Date: 2021-07-02 15:00+0200\n"
+"Last-Translator: \n"
+"Language-Team: \n"
+"Language: fr\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 3.0\n"
+
+#: src/clean.rs:31
+msgid "Do you want to remove ALL AUR packages from cache?"
+msgstr "Voulez-vous supprimer TOUS les paquets AUR du cache ?"
+
+#: src/clean.rs:33
+msgid "Do you want to remove all other AUR packages from cache?"
+msgstr "Voulez-vous supprimer les autres paquets AUR du cache ?"
+
+#: src/clean.rs:40
+msgid "Clone Directory: {}"
+msgstr "Dossier Clone: {}"
+
+#: src/clean.rs:46
+msgid ""
+"\n"
+"Diff Directory: {}"
+msgstr ""
+"\n"
+"Dossier Diff: {}"
+
+#: src/clean.rs:48
+msgid "Do you want to remove all saved diffs?"
+msgstr "Voulez-vous supprimer toutes les diffs sauvegardées ?"
+
+#: src/clean.rs:64
+msgid "can't open diff dir: {}"
+msgstr "impossible d'ouvrir le dossier diff: {}"
+
+#: src/clean.rs:75 src/clean.rs:133
+msgid "could not remove '{}'"
+msgstr "impossible de supprimer '{}'"
+
+#: src/clean.rs:93
+msgid "can't open clone dir: {}"
+msgstr "impossible d'ouvrir le dossier clone: {}"
+
+#: src/command_line.rs:127
+msgid "option {} expects a value"
+msgstr "l'option {} nécessite une valeur"
+
+#: src/command_line.rs:158 src/command_line.rs:327
+msgid "option {} does not allow a value"
+msgstr "option {} ne prend pas de valeur"
+
+#: src/command_line.rs:318
+msgid "unknown option --{}"
+msgstr "option inconnue --{}"
+
+#: src/command_line.rs:321
+msgid "unknown option -{}"
+msgstr "option inconnue-{}"
+
+#: src/completion.rs:27
+msgid "failed to open cache file '{}'"
+msgstr "erreur lors de l'ouverture du fichier de cache '{}'"
+
+#: src/completion.rs:65
+msgid "could not update aur cache"
+msgstr "impossible de mettre à jour le cache de l'AUR"
+
+#: src/config.rs:174
+#, fuzzy
+msgid "invalid value '{val}' for key '{key}', expected: {exp}"
+msgstr "valeur '{:val}' invalide pour la clé '{:key}', attendu: {:exp}"
```

Comment:
It's `{val}` not `{:val}`.

Reply:
Yes a PR has been opened on my fork. It will be fixed soon